Abhi-AnimeX/app/src/main/java/com/abhijith/animex/data/model/AnimeEntity.kt

```diff
@@ -0,0 +1,49 @@
+package com.abhijith.animex.data.model
+
+// temporary implementation to pass data from viewmodel instead of hardcoding it in composable
```

Comment:
a good practice in such cases, rather than saying `temporary implementation`, file issue and say `//TODO(issue_number)`